### DISCUSSION TITLE
FAIL - Application at context path /OpenClinica could not be started

### DISCUSSION THREAD

**Discussion by marwanada** [community/community#118572] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello Team,

Sorry if this is not a relevant topic to mention here. I am very new to databases hardly knowing my way through.

I have downloaded an open label software to my laptop (Openclinica) and whenever I start the app on tomcat I get the message “ FAIL - Application at context path /OpenClinica could not be started”

in Catalina log file mentions  “INFO: Illegal access: this web application instance has been stopped already”

Any clue what to do to run the application?

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @marwanada!

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Discussions category is solely related to conversations around the [GitHub product Discussions](https://docs.github.com/en/discussions/collaborating-with-your-community-using-discussions/about-discussions). This question should be in the General category. I've gone ahead and moved it for you. Good luck!
